modules/mobile/app/models/mobile/v0/payment_information.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module Mobile
+  module V0
+    class PaymentInformation < Common::Resource
+      attribute :id, Types::String
```

Comment:
I recently added the id to the enrollment status model as well. I initially tried keeping the id (which isn't from upstream) out of the model because it's not really a "payment info id", but that complicates the serializer because the serialized object needs an `id` method. As we move to requiring models to validate the data, I think this pattern of 
- fetch data
- adapt data
- create instance of model (with id)
- serialize
Keeps things easy to follow because all data changes should happen in the "adapt" phase and the model is a final representation of what will be serialized and ensures that everything that's serialized is also type validated. We should no longer need structs in our serializers, which I think is a bit of a relief.